vllm/entrypoints/openai/serving_chat.py

```diff
@@ -150,6 +150,7 @@ async def create_chat_completion(
                     conversation=conversation,
                     chat_template=request.chat_template or self.chat_template,
                     add_generation_prompt=request.add_generation_prompt,
+                    continue_final_message=request.continue_final_message,
```

Comment:
Let's update `apply_mistral_chat_template` to emit a warning that `continue_final_message` isn't supported (similar to how chat template is being handled).

Reply:
Done. I also added a warning when `add_generation_prompt` is provided, as it is also ignored.